### PR TITLE
:sparkles: Get Invite & Delete invite

### PR DESCRIPTION
**Pull request by Sigmanificient**
### Changes

-   `adds`: Get Invite & Delete invite

 ### Check off the following

-   [X] I have tested my changes with the current requirements
-   [X] My Code follows the pep8 code style.

### CODE REVIEW

File: pincer/objects/guild/guild.py [hunk 1]
```diff
@@ -1312,6 +1329,13 @@ async def delete_integration(
             headers={"X-Audit-Log-Reason": reason},
         )
 
+    async def delete_invite(self, code: str):
+        """|coro|
+        Deletes an invite.
+        Requires the ``MANAGE_GUILD`` permission.
```
Comment (Lunarmagpie) [inline pincer/objects/guild/guild.py:1335]:
```suggestion
        Requires the ``MANAGE_GUILD`` intent.
```

**Review by Lunarmagpie — APPROVED** [review]

File: pincer/objects/guild/guild.py [hunk 2]
```diff
@@ -1279,6 +1279,23 @@ async def get_invites(self) -> AsyncGenerator[Invite, None]:
         for invite_data in data:
             yield Invite.from_dict(invite_data)
 
+    async def get_invite(self, code: str) -> Invite:
+        """|coro|
+        Returns an invite object for an invite code.
```
Comment (trag1c) [inline pincer/objects/guild/guild.py:1284]:
```suggestion
        Returns an Invite object for the given invite code.
```

File: pincer/objects/guild/guild.py [hunk 3]
```diff
@@ -1279,6 +1279,23 @@ async def get_invites(self) -> AsyncGenerator[Invite, None]:
...
+
+        Parameters
+        ----------
+        code : :class:`str`
+            The invite code to get the invite for.
+
+        Returns
+        -------
+        :class:`~pincer.objects.invite.Invite`
```
Comment (trag1c) [inline pincer/objects/guild/guild.py:1293]:
```suggestion
        :class:`~pincer.objects.guild.invite.Invite`
```

File: pincer/objects/guild/guild.py [hunk 4]
```diff
@@ -1312,6 +1329,13 @@ async def delete_integration(
             headers={"X-Audit-Log-Reason": reason},
         )
 
+    async def delete_invite(self, code: str):
+        """|coro|
+        Deletes an invite.
+        Requires the ``MANAGE_GUILD`` intent.
+        """
```
Comment (trag1c) [inline pincer/objects/guild/guild.py:1336]: Missing parameters

**Review by trag1c — CHANGES_REQUESTED** [review]

**Review by trag1c — APPROVED** [review]

**Pull request merged**